### ISSUE TITLE
Cannot Read Property 'subustr' of undefined

### ISSUE THREAD

**Issue by fassetar** [fassetar/flavorstrap#18]
This happens when I create the src/dest files without a key value. 

```
 files:
               {
                   'src/bootstrap.scss','test/bootstrap.css'
               }
```

minor fix and probably can look at other modules for a solution.

**Comment by fassetar** [comment]
This fix is related to #20

**Comment by fassetar** [comment]
This bug will go away with the new package but will make note of it.
